package.json

```diff
@@ -16,6 +16,9 @@
       "types": "./lib/node/index.d.ts",
       "require": "./lib/node/index.js",
       "default": "./lib/node/index.mjs"
+    },
+    "./package.json": {
```

Comment:
What's the difference between this and 

```json
{
  "./package.json": "./package.json"
}
```

I've seen [other libraries](https://github.com/ReactiveX/rxjs/blob/630d2b009b5ae4e8f2a62d9740738c1ec317c2d5/package.json#L67) using a direct path, so I assume that's equivalent to "default". Do you happen to know if there's a difference?

Reply:
@kettanaito yep, when just a string, it's default.  When an object is passed it's evaluated top to bottom, returning the first match!

Reply:
@mattcosta7, your wizardry saves the realm once more! 🙇 Thanks, Matt.